libs/minilibx/hooks.md

```diff
@@ -69,6 +69,44 @@ a key. As you can see, we register a hook function with `mlx_key_hook`. However
 in the background it simply calls the function `mlx_hook` with the appropriate
 X11 event types. We will discuss this in the next chapter.
 
+## Hooking into mouse events
+
+Also, you can hook mouse events.
+
+```c
+mlx_mouse_hook(vars.win, mouse_hook, &vars);
+```
+
+<details>
+  <summary>Mouse buttons</summary>
+
+<table>
+<tr>
+<th align="center">Schema</th>
+<th align="center">Buttons</th>
+</tr>
+<tr>
+<td>
+
+![Mouse buttons](https://user-images.githubusercontent.com/92152391/147275282-90ff2fa2-810d-4c7d-b7bd-0c72d6192d7f.png)
```

Comment:
Please put this image in the actual repo and define your source. It needs to be credited accordingly.

Reply:
I took a picture of a random mouse on google (I think it was take by logitech for commercial use).
![image](https://user-images.githubusercontent.com/92152391/147289647-a6ca22ec-0f4c-434a-82cd-78620bfc2f8e.png)